analyze.py

```diff
@@ -392,7 +400,7 @@ def generate_obs_gap_yaml(triggering_points, path, project, node_ignore):
         yaml_map["se-namespace"] = effect["namespace"]
         yaml_map["se-rtype"] = effect["rtype"]
         yaml_map["se-etype"] = effect["etype"]
-        yaml_map["description"] = time_travel_description(yaml_map)
+        yaml_map["description"] = obs_gap_description(yaml_map)
```

Comment:
This looks hard coded to obs_gaps. You want the description to be generated based on the mode.

Reply:
Yes, there are two function `generate_obs_gap_yaml` and `generate_time_travel_yaml` for different mode, and `obs_gap_description` is inside  `generate_obs_gap_yaml`. I think there is some github's diff display issue, if you expand the code snippet, you can see the two function.